### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by ynrstp**
Added multiple styles and added a graphic for the readme, displaying all the available templates, including the ones made by @RyanFitzgerald.

### CODE REVIEW

File: dev/js/wow_wizard.js [hunk 1]
```diff
@@ -66,11 +66,71 @@
...
 					imageChoiceCircleBackgroundColor: "#A2B5EB"
+				},
+        peach: {
+					activeIndicatorBackgroundColor: '#DB764F',
+					activeIndicatorTextColor: '#FFF',
+					questionAndAnswerTextColor: '#7A2851',
+					circleColor: '#7A2851',
+					outlineColor: '#DB764F',
+					lineColor: '#DB764F',
+          activeButtonBackgroundColor: '#DB764F',
```
Comment (yigitozkavci) [inline dev/js/wow_wizard.js]: Indentation issue here

File: dev/js/wow_wizard.js [hunk 2]
```diff
@@ -66,11 +66,71 @@
...
+					activeButtonTextColor: '#FFF',
+					passiveButtonTextColor: '#000',
+					buttonTextColor: '#FFF',
+					imageChoiceBorderColor: "#5CD6FF",
+					imageChoiceCircleBackgroundColor: "#834820"
 				}
 			};
 			var styles = {
 				material: {
-					borderRadius: '0.2px'
```
Comment (yigitozkavci) [inline dev/js/wow_wizard.js:103]: For material design, we are not using too much border radius. What is this change good for?

**Review by yigitozkavci — CHANGES_REQUESTED** [review]

File: dev/js/wow_wizard.js (hunk 2 above)
Comment (ynrstp) [reply dev/js/wow_wizard.js:103]: I felt like the border-radius was barely visible at 0.2px so I increased it, but I'll change it back if that's what you want.

**Review by yigitozkavci — APPROVED** [review]

**Comment by yigitozkavci** [comment]
Thanks so much, it looks very good now :) Lastly, can you post screenshots for everyone to see new templates? @ynrstp

**Comment by ynrstp** [comment]
Added screenshots of the new templates to the img folder :)

cherry
coconut
passionfruit
peach

**Comment by yigitozkavci** [comment]
Wow, awesome work, loved the image you put on README.md, thanks! Merging :)

**Comment by yigitozkavci** [comment]
Your branch is not up to date, can you please pull, merge the conflicts then push again @ynrstp? If I do the merging, commit may not be shown as its done by you.

**Review by yigitozkavci — CHANGES_REQUESTED** [review]
You should first pull yigitozkavci:master branch, then merge these three files:
- `README.md`
- `dev/js/wow_wizard.js`
- `dist/js/wow_wizard.min.js`

**Comment by ynrstp** [comment]
Thanks for explaining, working on it now

**Comment by ynrstp** [comment]
I think I did it, can you check @yigitozkavci ?  Sorry it took so long

File: dev/js/wow_wizard.js [hunk 3]
```diff
@@ -36,68 +36,127 @@
...
+            circleColor: '#834820',
+            outlineColor: '#5CD6FF',
+            lineColor: '#5CD6FF',
+            activeButtonBackgroundColor: '#5CD6FF',
+            passiveButtonBackgroundColor: '#FFF',
+            activeButtonTextColor: '#FFF',
+            passiveButtonTextColor: '#000',
+            buttonTextColor: '#FFF',
+            imageChoiceBorderColor: "#5CD6FF",
+            imageChoiceCircleBackgroundColor: "#834820"
```
Comment (yigitozkavci) [inline dev/js/wow_wizard.js:128]: Missing closing bracket :)

**Review by yigitozkavci — CHANGES_REQUESTED** [review]

**Comment by yigitozkavci** [comment]
Merge conflicts are resolved, after that syntax bug is fixed, I can merge it @ynrstp

**Comment by yigitozkavci** [comment]
Approved and merged, thanks so much @ynrstp!

**Pull request merged**